service/controller/aws/cluster.go

```diff
@@ -206,23 +209,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 				v5ResourceSet,
 				v6ResourceSet,
 			},
-		}
-
-		resourceRouter, err = controller.NewResourceRouter(c)
-		if err != nil {
-			return nil, microerror.Mask(err)
-		}
-	}
-
-	var clusterController *controller.Controller
-	{
-		c := controller.Config{
-			CRD:            v1alpha1.NewAWSClusterConfigCRD(),
-			CRDClient:      crdClient,
-			Informer:       newInformer,
-			Logger:         config.Logger,
-			ResourceRouter: resourceRouter,
-			RESTClient:     config.G8sClient.ProviderV1alpha1().RESTClient(),
+			RESTClient: config.G8sClient.ProviderV1alpha1().RESTClient(),
```

Comment:
While changing this, could you maybe fix the client to use correct rest client? This should be:
`config.G8sClient.CoreV1alpha1().RESTClient()`. I know it was like that and it doesn't matter as both have decoders & stuff under the same configuration, but for the consistency still.

Reply:
Addressed in https://github.com/giantswarm/cluster-operator/pull/198.